bin/ns-status.js

```diff
@@ -3,10 +3,9 @@
 
 var os = require("os");
 
-var requireUtils = require('../lib/require-utils')
-  , safeRequire = requireUtils.safeRequire
-  , requireWithTimestamp = requireUtils.requireWithTimestamp
-  ;
```

Comment:
Putting the comma at the beginning of each new line (and the ; on its own line afterward) makes diffs cleaner when you add new variables to the list.

If you'd rather have a different `var foo = bar ;` statement instead of a comma-separated list, that would also work.

Reply:
OK will revert in a bit